ember-mobile-menu/src/components/mobile-menu.gjs

```diff
@@ -320,16 +309,24 @@ export default class MobileMenu extends Component {
 
   @action
   close(animate) {
+    if (!this.hasRendered) return;
+
     this.onClose(this, 0, animate);
   }
 
+  hasRendered = false;
+
   @action
-  openOrClose(open, animate = true) {
+  openOrClose(open) {
```

Comment:
You effectively removed a feature here by removing the animate argument. I think the main use case was fastboot though 🤔

Reply:
Based on the yields, and all the arguments passed places, it wasn't public api

https://github.com/nickschot/ember-mobile-menu/pull/1160/files/c1fd761c79667f45694705f6fed1ac48a2171f67#diff-ad10e35109ce6436c70d0e587ca124212e360bac5b6272f9257d31cae545429bL382